connector/src/main/java/com/google/cloud/spark/bigquery/converters/ArrowSchemaConverter.java

```diff
@@ -0,0 +1,497 @@
+package com.google.cloud.spark.bigquery.converters;
```

Comment:
Add a license

Reply:
Update the [LICENSE file](https://github.com/GoogleCloudDataproc/spark-bigquery-connector/blob/master/LICENSE) as well to add specific reference to spark code.  As an example look at the [Arrow LICENSE file](https://github.com/apache/arrow/blob/master/LICENSE.txt#L210) and how it does attribution.

Reply:
Arrow is also under Apache License 2.0 which we already include in our project.
I am not exactly sure about what  exactly I am supposed to change here. @davidrabinowitz  could you advise accordingly?

Reply:
I provided the Arrow file as an example of how to provide attribution to code that is copied from other projects under the apache 2.0 license file.  I think you should add an additional section to the existing license file that acknowledges code from spark was used from spark.